pkgs/os-specific/darwin/apple-sdk-11.0/libSystem.nix

```diff
@@ -1,4 +1,4 @@
-{ stdenvNoCC, buildPackages, MacOSX-SDK }:
+{ stdenvNoCC, buildPackages, MacOSX-SDK, withLibresolv ? null }:
```

Comment:
It simplifies the definition of the bootstrap stdenv. It needs to override Libsystem not to include Csu and libresolv. It’s easier and cleaner if I can do that unconditionally (especially since Libsystem is versioned inconsistently depending on whether it comes from the SDK or is built from a source release).

Reply:
Maybe we should add a comment explaining the reasoning?
```suggestion
{ stdenvNoCC, buildPackages, MacOSX-SDK
  # Enables unconditional libSystem override
, withLibresolv ? null
}:
```